packages/actor-query-operation-group/lib/ActorQueryOperationGroup.ts

```diff
@@ -0,0 +1,131 @@
+import { ArrayIterator } from 'asynciterator';
+import { Map, Set } from "immutable";
+import { termToString } from 'rdf-string';
+import { Algebra } from "sparqlalgebrajs";
+import { AggregateEvaluator } from 'sparqlee';
+
+import {
+  ActorQueryOperation,
+  ActorQueryOperationTypedMediated,
+  Bindings,
+  IActorQueryOperationOutputBindings,
+  IActorQueryOperationTypedMediatedArgs,
+} from "@comunica/bus-query-operation";
+import { ActionContext, IActorTest } from "@comunica/core";
+
+/**
+ * A comunica Group Query Operation Actor.
+ */
+export class ActorQueryOperationGroup extends ActorQueryOperationTypedMediated<Algebra.Group> {
+
+  constructor(args: IActorQueryOperationTypedMediatedArgs) {
+    super(args, 'group');
+  }
+
+  public async testOperation(pattern: Algebra.Group, context: ActionContext): Promise<IActorTest> {
+    pattern.aggregates.forEach((aggregate) => {
```

Comment:
I would suggest using the [_for/in_](https://www.w3schools.com/jsref/jsref_forin.asp) statement here. (Repeated lambda creations cause a slight overhead in V8)